docs/source/user-guide/quickstart/index.rst

```diff
@@ -10,7 +10,6 @@ Prerequisite Software
   :includehidden:
 
   install
-  environment
```

Comment:
Same request here as in `/contribute/quickstart/index.rst`

Reply:
Done